### ISSUE TITLE
[wg/browser-tools-testing] Browser Testing and Tools Working Group Charter

### ISSUE THREAD

**Issue by tidoust** [w3c/strategy#558]
New charter proposal, reviewers please take note.

# Charter Review

[Charter](https://w3c.github.io/charter-drafts/2026/charter-wg-btt.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fcharter-template.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2026%2Fcharter-wg-btt.html)

Expected end of charter refinement phase: [2026-06-19](https://lists.w3.org/Archives/Member/w3c-ac-members/2026AprJun/0037.html)

If applicable:

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2024%2Fbtt-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2026%2Fcharter-wg-btt.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/browser-tools-testing)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [X] Existing WG recharter 
 
Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/orgs/w3c/projects/97/views/2).**

## Substantive changes

No substantive change. Same scope and list of deliverables.

## Known or potential areas of concern

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

The [charter-drafts repository](https://github.com/w3c/charter-drafts/)

The Working Group is not considering substantive changes to its scope, but is currently assessing support for the [AT Driver](https://w3c.github.io/at-driver/) deliverable, which has not been published on the Recommendation track during this chartering period. The deliverable may be dropped from the draft charter as a result.

## Anything else we should think about as we review? 

Cc @AutomatedTester

Note: The Technical Strategy Team Lead or Project & Process Team Lead will assign the issue to a [Charter Facilitator](https://www.w3.org/guide/process/charter.html#cr-acceptance) for new charters. For rechartering, the team contact is the Charter Facilitator by default, please assign the issue to them directly.

**Comment by plehegar** [comment]
Please, start the charter refinement phase.

**Comment by plehegar** [comment]
[Charter refinement](https://lists.w3.org/Archives/Member/w3c-ac-members/2026AprJun/0037.html)

**Comment by ruoxiran** [comment]
no comment or request from APA.

**Comment by ruoxiran** [comment]
@daniel-montalvo, any thoughts from you or ARIA WG ?  concerning "The Working Group is not considering substantive changes to its scope, but is currently assessing support for the [AT Driver](https://w3c.github.io/at-driver/) deliverable, which has not been published on the Recommendation track during this chartering period. The deliverable may be dropped from the draft charter as a result." cc: @iadawn

**Comment by daniel-montalvo** [comment]
Thanks @ruoxiran for bringing this to my attention.

I'd suggest if the Browser and Testing WG doesn't feel like  they are going to be able to progress with AT Driver, at least leave the door open to a potential collaboration with the ARIA Working Group.
 If dropping the spec completely, I'd be interested to pitch with ARIA the idea of picking it up as a joint deliverable.

**Comment by AutomatedTester** [comment]
> I'd suggest if the Browser and Testing WG doesn't feel like they are going to be able to progress with AT Driver, at least leave the door open to a potential collaboration with the ARIA Working Group. If dropping the spec completely, I'd be interested to pitch with ARIA the idea of picking it up as a joint deliverable.

This is another option we can do. @jugglinmike, this could be another option for you to keep working on AT Driver. The need is there, it's just how much we, BTT, can help you deliver on this and if ARIA want's a joint deliverable...

**Comment by himorin** [comment]
No comment or request from i18n

**Comment by tidoust** [comment]
@simoneonofri, @tjwhalen, @ylafon, may I consider that the Security, Privacy and TAG reviews have stalled on this one and proceed to the next chartering step?

**Comment by tidoust** [comment]
The plan discussed with the ARIA WG is to drop the AT Driver from the Browser Testing and Tools WG and to have the ARIA WG take it up next time it recharters. The document had not been published on the Recommendation track yet, no need for a more formal transition.

I updated the draft charter to drop AT Driver, as well as to add the ARIA WG to he coordination list to set the expectation that the Browser Testing and Tools WG will coordinate with the ARIA WG, e.g., to align the spec with WebDriver BiDi concepts.

**Comment by simoneonofri** [comment]
@tidoust We discussed during the SING meeting, and there are no blocking comments, just some notes:
- There is a change in the Success Criteria for security and privacy consideration section from "will" to "should", but this is a delta from the new charter template
- [not Security] While the mission now includes debugging and troubleshooting, this is not reflected in the scope section.

So, no blockers from Security.

**Comment by tidoust** [comment]
> [not Security] While the mission now includes debugging and troubleshooting, this is not reflected in the scope section.

Interestingly, that change got introduced by the charter assistant, which reuses the mission description as it appears on the [group's home page](https://www.w3.org/groups/wg/browser-tools-testing/), which has the additional words for some reason. I had failed to notice that change until now. I'm fine with it, it seems to be a better description of what testing automation encompasses.

**Comment by matatk** [comment]
Given [the plan for ARIA to adopt the AT-Driver work that this WG is dropping](https://github.com/w3c/strategy/issues/558#issuecomment-5089104356) that @tidoust mentioned above, APA has no concerns.

**Comment by plehegar** [comment]
[VOTE by 2026-09-18: Proposed Charter for the Browser Testing and Tools Working Group](https://lists.w3.org/Archives/Member/w3c-ac-members/2026JulSep/0044.html)
